quipucords/api/credential/model.py

```diff
@@ -39,6 +39,7 @@ class Credential(models.Model):
     password = models.CharField(max_length=1024, null=True)
     auth_token = models.CharField(max_length=1024, null=True)
     ssh_keyfile = models.CharField(max_length=1024, null=True)
+    ssh_keyvalue = models.CharField(max_length=65536, null=True)
```

Comment:
I'm thinking about this `max_length`. Is 64K large enough for 99% of possible keys? I recall you mentioned earlier that you tested using a large key, but I don't recall exactly how big it was.

Internally, I believe this is a `VARCHAR` at the DB level, and Postgres is smart enough to store only what is needed. So, setting an absurdly large maximum length _should_ be safe as long as users aren't deliberately abusing the feature.

Reply:
VARCHAR maps to Django's TextField, right? I was going to ask if we couldn't simply use that 🤔

Reply:
Yes, the CharField are Varchars internally and PG only allocates what it needs. I think I've tackled the largest (16kbit rsa), 
 some info in this comment: https://github.com/quipucords/quipucords/pull/2390/commits/da45b52ea7d860632953980d933732e71122febc.  No preference CharField vs TextField (both Varchars). The docs do mention using CharField for smaller data sizes. We could use a TextField with no max_length. Although we should still set an upper limit so that users don't abuse it.

Reply:
Hmm, I did a Migration to TextField, even though max_length is taken by Django, this translate to just text in Postgres with no limit (other than the DB's 1GB limit).

I verified this by doing a Migration to TextField with the max_length set to 65536 and I had it here. DB just shows `text`:

<img width="638" alt="image" src="https://github.com/quipucords/quipucords/assets/5797328/76bbc494-6097-4e5d-8fea-765de8a39be5">

And the following proving it's wide open:

```
In [3]: c = Credential.objects.get(id=26)

In [4]: c.ssh_keyvalue = "1234567890" * 100000

In [5]: c.save()
```

```
In [6]: d = Credential.objects.get(id=26)

In [7]: len(d.ssh_keyvalue)
Out[7]: 4050355
```

So, yea, here's a potential abuse of someone feeding 1MB in and Ansible encrypting bumping it up by 4. 

We need that max_length for our sanity so we're safer to stick with CharField.